### PR TITLE
Projecthome

### PR DESCRIPTION
**Pull request by Erwan60**

### CODE REVIEW

File: client/src/components/Project/project.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { useState, prompt} from "react";
+import "./project.css";
+import sproject from "../../assets/images/sproject.png";
+
+function Project() {
+  const [project, setProject] = useState([
```
Comment (Brian-boudrioux) [inline client/src/components/Project/project.jsx]: Dont use static data, you should fetch your api in a useEffect and update your state with the data returned by your api

File: client/src/components/Project/project.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "Horizon Nexus",
+    "Zenith Path",
+    "Quantum Leap",
+    "Celestial Haven",
+    "Cobalt Wave",
+    "Veridian Pulse",
+  ]);
+
+  const addProject = () => {
+    const newProject = prompt("Entrez le nom du nouveau projet :");
```
Comment (Brian-boudrioux) [inline client/src/components/Project/project.jsx]: prompt ? you need to respect the graphical chart defined in your figma !

File: server/app/routers/api/router.js [hunk 3]
```diff
@@ -7,10 +7,10 @@ const router = express.Router();
 /* ************************************************************************* */
 
 const usersRoutes = require("./users/router"); // Management des utilisateurs
-const tablesRoutes = require("./tables/router"); // Management des tableaux
```
Comment (Brian-boudrioux) [inline server/app/routers/api/router.js:10]: Why did you deleted tablesRoutes ? maybe that can affect a other feature of the project, you should not touch any code that in not in concern with your user story

**Review by Brian-boudrioux — CHANGES_REQUESTED** [review]
see the comments above, your feature isnt finish !

**Review by Alexandre78R — APPROVED** [review]

**Pull request merged**